### PR TITLE
set objectives for centralized workflow

### PR DESCRIPTION
**Pull request by annefou**

### CODE REVIEW

**Comment by annefou** [comment]
Working on centralized workflow:
- Start showing the proper worflow (create a branch, make a PR)
- Simplify graphics
- Update your local `master` branch and add an exercise (may be optional) to discuss problems when not using branching.

File: _episodes/02-centralized.md [hunk 1]
```diff
@@ -88,128 +150,75 @@ $ git commit
...
+Can we leave out the `-u`?
 
+### 6. Submit a pull request
 
-### 6. **Stop here** and discuss why push for most participants was rejected
+Submit a pull request from your branch towards the `master` branch.
+Do this through the web interface.
 
-You probably see something like this:
+A pull-request means: "please review my changes and if you agree, merge them with a mouse-click".
```
Comment (rkdarst) [inline _episodes/02-centralized.md]:
```suggestion
A pull-request means: "please review my changes and if you agree, merge them with a mouse-click". In a popular project, it means that anyone can
contribute with *almost no work* on the maintainer's side - a big win.
```
This is an opinion that maybe doesn't need to be here (I'm not sure if it's somewhere else already), but at some point it's worth pointing out that it makes contributions easy - unlike ages ago.

File: _episodes/02-centralized.md [hunk 2]
```diff
@@ -71,13 +123,23 @@ This is a representation of what happens when you clone:
 $ cd centralized-workflow-exercise
 ```
 
+### 3. Create a branch `yourname/somefeature` pointing at your commit
 
-### 3. Create a file with a unique name, e.g.: `yourusername.txt`
+Create a branch from the current `master`:
+
+```
+$ git branch yourname/somefeature
```
Comment (rkdarst) [inline _episodes/02-centralized.md]:
```suggestion
$ git branch yourname/somefeature
$ git checkout yourname/somefeature
```

**Comment by bast** [comment]
> But... I had heard something in that discussion about having people see successful regular pushes first, before going to the pull requests. I remembered that was mentioned as one of the points of small groups - so that centralized-with-push works and they can see a typical easy case before the abstract PRs. With small groups, we can even give them five minutes to practice committing, pushing, experimenting, and discussing to see what works and doesn't work. I think this small break time. Then, go on to pull requests.

I agree, followed up in #118.

**Pull request merged**